compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -825,6 +825,7 @@ private void defineServiceDeclarations(DataInputStream inputStream) throws IOExc
     }
 
     private void definePackageLevelVariables(DataInputStream dataInStream) throws IOException {
+        Location pos = readPosition(dataInStream);
```

Comment:
We can probably add a BIR test for this too. e.g., https://github.com/ballerina-platform/ballerina-lang/pull/36457/files#diff-159125135700a3223283f43c74293e9213721379205be2e3fe12fe2af28b3883R477 onward.

Reply:
Added a test